src/api/client.js

```diff
@@ -18,10 +25,31 @@ function assetTypesFromState(assetTypes) {
   return Object.keys(assetTypes).filter(key => assetTypes[key]);
 }
 
-export function requestAssets(courseId, { page = 0, pageSize = 50, sort = 'sort', assetTypes = {} }) {
```

Comment:
The API has these same defaults[ built in](https://github.com/edx/edx-platform/blob/master/cms/djangoapps/contentstore/views/assets.py#L32), so I didn't want them specified here as well.